Documentation/api.md

```diff
@@ -380,7 +380,7 @@ The `NodeExporterConfig` resource defines settings for the `node-exporter` agent
 | Property | Type | Description |
 | -------- | ---- | ----------- |
 | collectors | [NodeExporterCollectorConfig](#nodeexportercollectorconfig) | Defines which collectors are enabled and their additional configuration parameters. |
-| maxProcs | uint32 | The target number of CPUs on which the Node Exporter's process will run. The default value here is `0`, which means that Node Exporter runs on all CPUs. If a kernel deadlock occurs or if performance degrades when reading from `sysfs` concurrently, you can change this value to `1`, which limits Node Exporter to running on one CPU. For nodes with a high CPU count, you can set the limit to a low number, which  saves resources by preventing Go routines from being scheduled to run on all CPUs. However, I/O performance degrades if the `maxProcs` value is set too low, and there are many metrics to collect. |
+| maxProcs | uint32 | The target number of CPUs on which the Node Exporter's process will run. Use this to override the  default value which is set to the minimum of `4` and the number of CPU on the host. The default value is computed at runtime and set via the `GOMAXPROCS` environment variable before launching Node Exporter. If a kernel deadlock occurs or if performance degrades when reading from `sysfs` concurrently, you can change this value to `1`, which limits Node Exporter to running on one CPU. For nodes with a high CPU count, setting the limit to a low number saves resources by preventing Go routines from being scheduled to run on all CPUs. However, I/O performance degrades if the `maxProcs` value is set too low, and there are many metrics to collect. |
```

Comment:
> the minimum of `4` and the number of CPU on the host

The wording confused me a bit. Maybe `to 4 or the number of CPUs on the host, whichever is smaller.`

Reply:
Done.
(Know that I thought about using min(4, xxx) at the beginning haha)

Reply:
haha, anyway let's wait for @bburt-rh inputs :)

Reply:
@simonpasquier @machine424 - I added some suggested changes.